APSE-System/Immerscale
# task57: implement endpoint for receiving images

- all images are returned which correspond to a given project ID

how to test:
- make sure there are images in your database
- check the API endpoint `localhost:8080/workerView/images?id=1`
